### PR TITLE
chore: add volunteer DominicFrei 

### PR DESCRIPTION
**Pull request by DominicFrei**

### CODE REVIEW

File: data/users/Dominic_Frei.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "name": "Dominic Frei",
+  "bio": "Senior Software Engineer && Developer Advocate @MongoDB",
+  "image": "https://pbs.twimg.com/profile_images/1321489002759946240/J2aAULS-_400x400.jpg",
+  "categories": [
+    {
+      "slug": "programming languages",
+      "topics": ["C#", "Python"]
+    },
+    {
+      "slug": "IDEs",
```
Comment (codewithvoid) [inline data/users/Dominic_Frei.json:11]: we don't have category for IDEs so will need to drop this :|

File: data/users/Dominic_Frei.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      "topics": ["Jetbrains Rider", "Jetbrains PyCharm", "Unity3D"]
+    },
+    {
+      "slug": "areas",
+      "topics": ["backend development", "game development", "database development"]
```
Comment (codewithvoid) [inline data/users/Dominic_Frei.json:16]: you can mention more specific topics in the categories file above..these are pretty broad.

**Review by codewithvoid — CHANGES_REQUESTED** [review]
Thanks for volunteering @DominicFrei. Requested some changes.

**Comment by DominicFrei** [comment]
@codewithvoid Thank you for the quick review. I've added another commit to address your comments. :)

File: data/users/Dominic_Frei.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Dominic Frei",
+  "bio": "Senior Software Engineer @MongoDB",
```
Comment (codewithvoid) [inline data/users/Dominic_Frei.json:3]: nit: Can drop @MongoDB

File: data/users/Dominic_Frei.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  "image": "https://pbs.twimg.com/profile_images/1321489002759946240/J2aAULS-_400x400.jpg",
+  "categories": [
+    {
+      "slug": "programming",
+      "topics": ["C#", "Python"]
+    },
+    {
+      "slug": "software-development",
+      "topics": ["Jetbrains Rider", "Jetbrains PyCharm", "Unity3D"]
```
Comment (codewithvoid) [inline data/users/Dominic_Frei.json:12]: remove Jetbrains prefix from the topics.

File: data/users/Dominic_Frei.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    },
+    {
+      "slug": "database-development",
+      "topics": ["MongoDB"]
+    }
+  ],
+  "contacts": {
+    "twitter": "dominicfrei"
```
Comment (codewithvoid) [inline data/users/Dominic_Frei.json:20]: You can mention your discord id too. E.g
```json
"discord": "codewithvoid#8967"
```

File: data/users/Dominic_Frei.json [hunk 6]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Dominic Frei",
+  "bio": "Senior Software Engineer @MongoDB",
+  "image": "https://pbs.twimg.com/profile_images/1321489002759946240/J2aAULS-_400x400.jpg",
```
Comment (codewithvoid) [inline data/users/Dominic_Frei.json:4]: mind using github image url github image url
https://avatars.githubusercontent.com/u/1942012?v=4

It renders well across all regions

**Review by codewithvoid — CHANGES_REQUESTED** [review]

File: data/users/Dominic_Frei.json (hunk 3 above)
Comment (codewithvoid) [reply data/users/Dominic_Frei.json:3]: Optional.

Was wondering if its too long for card title. Will create separate issue if that's the case.

**Review by codewithvoid — APPROVED** [review]
Thanks for making the changes. Welcome!

**Comment by DominicFrei** [comment]
Thank you, @codewithvoid !

**Pull request merged**